src/Utils.vala

```diff
@@ -80,4 +80,23 @@ namespace Display.Utils {
 
         return min_scale;
     }
+
+    public static bool is_night_light_supported () {
+        try {
+            string standard_output;
+            int exit_status;
+            Process.spawn_command_line_sync ("/usr/bin/uname -p",
```

Comment:
If the intention is to only check if we're running on 64bit, a much simpler check of the size of a void pointer `if (sizeof (void*) == 8)` will suffice.

Reply:
Nope, the intention is to check ARM or x86, they can both be either 64-bit or 32-bit.

Reply:
Alright, good to know.